source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,203 @@
+.. _minio-gateway-migration:
+
+=======================================
+Migrate from Gateway or Filesystem Mode
+=======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Background
+----------
+
+The MinIO Gateway and the related filesystem mode entered a feature freeze in July 2020.
+In February 2022, MinIO announced the `deprecation of the MinIO Gateway <https://blog.min.io/deprecation-of-the-minio-gateway/?ref=docs>`__.
+Along with the deprecation announcement, MinIO also announced that the feature would be removed in six months time.
+
+As of :minio-release:`RELEASE.2022-10-29T06-21-33Z`, the MinIO Gateway and the related filesystem mode code have been removed.
+Deployments still using the `standalone` or `filesystem` MinIO modes that upgrade to :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later receive an error when attempting to start MinIO.
+
+Overview
+--------
+
+To upgrade to a :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later, those who were using the `standalone` or `filesystem` deployment modes must create a new :ref:`Single-Node Single-Drive <minio-snsd>` deployment and migrate settings and content to the new deployment.
+
+This document outlines the steps required to successfully launch and migrate to a new deployment.
+
+.. important:: 
+
+   Standalone/file system mode continues to work on any release up to and including `RELEASE.2022-10-24T18-35-07Z <https://github.com/minio/minio/releases/tag/RELEASE.2022-10-24T18-35-07Z>`__.
+   To continue using a standalone deployment, install that MinIO release or any `earlier release <https://github.com/minio/minio/releases>`__.
+
+   Do not upgrade to any release later than ``RELEASE.2022-10-24T18-35-07Z``.
+
+Procedure
+---------
+
+#. Retrieve the existing deployment's **environment variables**
```

Comment:
I think we can tab this out and note as a pre-graph something like

"You can set MinIO configuration settings in environment variables and using :mc-cmd:`mc admin config set <mc admin config>`. 
Depending on your current deployment setup, you may need to retrieve the values for both:"

For envvars they can use something like `env | grep MINIO_" or check the `/etc/default/minio` file for systemd setups. Ultimately users can do whatever so general guidance here is sufficient.

Reply:
Moved to a note.